app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,33 @@
+class PasswordResetsController < ApplicationController
+  before_filter :verify_user, only: [:edit, :update]
+
+  def create
+    if user = User.find_by(email: params[:email].downcase)
+      UserMailer.reset_password_email(user)
+    end
+
+    redirect_to [:new, :password_reset], alert: "Instructions for resetting your password have been sent."
```

Comment:
we should try and keep lines below 80 chars whenever we reasonably can, and never longer than say 86 chars in any circumstance. The preceding line could be written like this:

```ruby
redirect_to [:new, :password_reset],
  alert: "Instructions for resetting your password have been sent."
```